monky.el

```diff
@@ -2882,6 +2890,19 @@ With a non numeric prefix ARG, show all entries"
       (error "Nothing staged")
     (monky-pop-to-log-edit 'commit)))
 
+(defun monky-commit-amend ()
+  "Amends previous commit.
+Brings up a buffer to allow editing of commit message."
+  (interactive)
+  (if (not (or monky-staged-files (monky-merge-p)))
```

Comment:
is this necessary? won't this prevent the case where you just want to edit the last commit message